api/src/test/java/org/openmrs/api/db/hibernate/HibernateDiagnosisDAOTest.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.api.db.hibernate;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.openmrs.Diagnosis;
+import org.openmrs.api.db.DiagnosisDAO;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class HibernateDiagnosisDAOTest extends BaseModuleContextSensitiveTest {
```

Comment:
This is the cause of your error. The base class should be BaseContextSensitiveTest
The one that you have is only used in modules.

Reply:
Noted